cloudformation/cid-crdc-resources.yaml

```diff
@@ -0,0 +1,709 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: "Deployment of AWS Config Resource Compliance Dashboard (CRCD) v2"
+
+Metadata:
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+      -
+        Label:
+          default: "QuickSight"
+        Parameters:
+            - PQuickSightUser
+      -
+        Label:
+          default: "AWS Config Logging"
+        Parameters:
+          - PConfigLoggingBucket
+          - PConfigLoggingAccountID 
+      -
+        Label:
+          default: "Athena Configuration (DO NOT CHANGE)"
+        Parameters:
+          - PAthenaWorkgroup
+          - PAthenaDatabase
+          - PAthenaTable
+          - PAthenaQueryResultBucketPrefix
+      
+    ParameterLabels:
+      PAthenaWorkgroup:
+        default: "Athena workgroup"
+      PAthenaDatabase:
+        default: "Athena database"
+      PAthenaQueryResultBucketPrefix:
+        default: "Prefix of the Athena query results bucket"
+      PAthenaTable:
+        default: "Athena table for AWS Config data"
+      PConfigLoggingBucket:
+        default: "AWS Config bucket"
+      PConfigLoggingAccountID:
+        default: "Config bucket account"
+      PQuickSightUser:
+        default: "User name of QuickSight user (as displayed in QuickSight admin panel)"
+      
+Parameters:
+  PQuickSightUser:
+    Type: String
+    MinLength: 1
+    Default: "Replace with QuickSight user"
+    Description: See https://quicksight.aws.amazon.com/sn/admin#users
+
+  PConfigLoggingBucket:
+    Type: "String"
+    Default: "your-config-bucket"
+    Description: "Name of the Amazon S3 bucket that collects AWS Config data (Required)"
+    AllowedPattern: '^[a-z0-9][a-z0-9-]{1,61}[a-z0-9]$'
+    ConstraintDescription: "Required"
+  
+  PConfigLoggingAccountID:
+    Type: "String"
+    Default: "123412341234"
```

Comment:
Same here, better not providing default here to avoid customer just accept it as is

Reply:
done and resolving as agreed with Iakov